### PR TITLE
TOOLS-2461 upgrade Manta/Triton 19.[12] components to 19.4

### PR DESCRIPTION
**Pull request by timfoster**

### CODE REVIEW

**Comment by papertigers** [comment]
@timfoster oh good it looks like the tests ran as a part of this PR 👍

File: cfwlogd/src/main.rs [hunk 1]
```diff
@@ -242,7 +242,7 @@ fn validate_log_files(vmobjs: &Vmobjs) {
                 Ok(info) => {
                     // If we find the "\n" we add 1 to the index so that when we truncate the file
                     // we keep the newline.
-                    let idx = info.index.and_then(|i| Some(i + 1)).unwrap_or(0);
+                    let idx = info.index.map(|i| i + 1).unwrap_or(0);
```
Comment (papertigers) [inline cfwlogd/src/main.rs:245]: This looks correct to me.  Map will map the value if the Option is Some otherwise it will leave it as None.

**Review by papertigers — APPROVED** [review]

File: cfwlogd/src/main.rs (hunk 1 above)
Comment (papertigers) [reply cfwlogd/src/main.rs:245]: Could also be cleaned up to use `info.index.map_or(0, |i| i +1)`

Comment (timfoster) [reply cfwlogd/src/main.rs:245]: Would you prefer that? I really just went with the clippy advice here?

**Pull request merged**